homu/server.py

```diff
@@ -39,7 +40,8 @@ def find_state(sha):
 def get_repo(repo_label, repo_cfg):
     repo = g.repos[repo_label].gh
     if not repo:
-        g.repos[repo_label] = repo = g.gh.repository(repo_cfg['owner'], repo_cfg['name'])
+        g.repos[repo_label] = repo = g.gh.repository(repo_cfg['owner'],
```

Comment:
Whoops, this double assignment thingy bugged me too the first time I looked at it and then I split it totally wrong the second time. Thanks for pointing that out!